unit-tests/contracts/Gaia.cdc

```diff
@@ -0,0 +1,846 @@
+import NonFungibleToken from "./NonFungibleToken.cdc"
```

Comment:
Why did you duplicated the contract files to your directory, the contracts should be the same for Flow Tokens or Dapper Utility Coins. Remove the contracts from your local directory and use the existing ones.

Reply:
Done